### DISCUSSION TITLE
6.1.8 will soon celebrate one year

### DISCUSSION THREAD

**Discussion by 0323pin** [grtcdr/macchina#300] [General]
Hi @grtcdr 
Long time, no see.

Happy new year to you and everyone involved in `macchina`.
It came to my attention that v6.1.8 will soon be celebrating one year. It caught my eye that there are a few recent commits and I've decided to ask ... is a new release, with updated dependencies, planned?

No rush but, you know the Rust world moves fast and one year is ... long :)
BTW, we are still on Rust-1.73 due to a branch freeze. I hope we move to 1.74.1 soon and I can move to 1.75 also soon.

Anyway, regards and looking forward to an update of my favorite fetcher.

**Comment by grtcdr** [comment]
Hi pin, happy new year! It's really good to hear from you. Yeah, it's been a while!

Wow, things must look quite stale from the outside, I can't believe it's been a year already... I've been slowly upgrading dependencies and fixing regressions as they uncover themselves. I've been taking my time with the upcoming release (that is v6.2.0) making sure to eliminate as many bugs as possible, and staying on top of a rapidly-changing set of dependencies, as you say, hasn't been so trivial.

Things are coming along, though, but at an admittedly slow pace.

Cheers!

**Comment by 0323pin** [comment]
> I've been taking my time with the upcoming release (that is v6.2.0) making sure to eliminate as many bugs as possible ... Things are coming along, though, but at an admittedly slow pace.

Cool, looking forward to it 👍

**Comment by 0323pin** [comment]
If you are bumping the MSRV to >= 1.70, may I suggest dropping the `atty` dependency and use the [IsTerminal](https://doc.rust-lang.org/stable/std/io/trait.IsTerminal.html) trait in the Rust standard library instead. `atty` is unmaintained and `cargo-audit` complains about it.
